packages/react-app/src/services/giveawayparticipation.ts

```diff
@@ -90,15 +95,7 @@ const meetRequirements = async (
     if (!accepted) return true;
 
     const location = await FrontendApiClient.getLocation(locationId);
-    if (
-      !location ||
-      !(await GeolocationService.isWithinRadius(
-        location.latitude,
-        location.longitude,
-        location.radius
-      ))
-    )
-      return false;
```

Comment:
I think this code should be kept, the idea is to be able to participate only when we cannot access/validate the location of the user. If we can access the user location if he is not in the range, then he should not be able to participate